node/src/main.rs

```diff
@@ -108,7 +110,7 @@ fn async_main() -> impl Future<Item = (), Error = ()> + Send + 'static {
     let ethereum_ws = matches.value_of("ethereum-ws");
 
     let ipfs_socket_addr = SocketAddr::from_str(matches.value_of("ipfs").unwrap())
-        .expect("could not parse IPFS address, expected format is host:port");
+        .expect("could not parse IPFS address, expected format is ipaddress:port");
```

Comment:
I think host is fine here.

Reply:
A `SocketAddr` is an IP address + port, it doesn't do any DNS lookups, see here: https://doc.rust-lang.org/std/net/enum.SocketAddr.html

I could add support for that though!